README.md

```diff
@@ -17,10 +17,5 @@ nosetests
 ```
 
 ### Etiquette
-
-- If you are gone for a week or two, please submit a PR to remove yourself from the `config.yml` file.
-- Feel free to change whether you belong in the `enchantress` or `hobbit` camp. `hobbits` are generally newer
-  while `enchantresses` are generally more experienced. Try and keep the two lists roughly equal. The script 
```

Comment:
Is there any reason to keep two lists (`list_one` and `list_two`) vs a single list if this documentation is removed and we are only using one list anyway? The distinction between the new list names seems even less obvious than before: hobbit == newer and enchantress == more experienced.

Reply:
Yeah that is a fair point. I was generalizing it thinking the use case of one list being less experienced and the other being more experienced only reflected one way you could use separate lists. Another example would be something like cross divisional pairing, so SaaS devs in one list and USH in another (obviously in this case we would probably want to support even more lists for solutions and india, but you get the point).

All that to say, our only use case at the moment is one list 🤷🏻